samples/canopen/canopen_sample/TestCanNode.hpp

```diff
@@ -75,30 +75,30 @@ class TestCanNode {
      */
     CO_OBJ_T objectDictionary[OBJECT_DICTIONARY_SIZE + 1] = {
         // Sync ID, defaults to 0x80
-        {CO_KEY(0x1005, 0, CO_UNSIGNED32 | CO_OBJ_D__R_), 0, (uintptr_t) 0x80},
+        {CO_KEY(0x1005, 0, CO_OBJ_DN__R_), 0, (uintptr_t) 0x80},
```

Comment:
This sample appears to be missing the "Mandatory Identification Keys" included in the other samples.